### DISCUSSION TITLE
SSR, 200.html, dynamic routes?

### DISCUSSION THREAD

**Discussion by edwh** [nuxt/nuxt#17504] [Questions]
In Nuxt2, generate produced a 200.html file which you could configure a web server to redirect 404 errors to, to handle dynamic routes.

I can't find this find in Nuxt3.  Has this changed?  Not been implemented yet?

Apologies if this is in the docs and I've not found it yet.

**Comment by liffiton** [comment]
I'm new to Nuxt, so I might be missing something, too, but it looks like those are generated if you set `ssr: false` in your Nuxt config.  In my app, that creates `.output/public/200/index.html` and `.output/public/404/index.html`.  They're identical to the main `index.html` it puts in `.output/public` in that case.  It looks like an empty shell that loads Nuxt JS and renders everything client-side.  But then that's all Nuxt does, and it doesn't pre-render anything.

I haven't figured out how to get Nuxt3 to pre-render the routes that it can *and* output that empty shell for routes it cannot.  Possibly one could just copy the `index.html` it generates with `ssr: false` and it for 404s?

**Comment by edwh** [comment]
That might indeed work, I've not tried.

It's been transferred to the issues list in https://github.com/nuxt/framework/issues/5003, so presumably the magic open source fairies will very kindly fix it.
